src/flask_track_usage/storage/__init__.py

```diff
@@ -47,7 +49,24 @@ def __init__(self, *args, **kwargs):
            - `args`: All non-keyword arguments.
            - `kwargs`: All keyword arguments.
         """
+        if "hooks" in kwargs:
+            self._temp_hooks = kwargs["hooks"]
```

Comment:
Why expose ``_temp_hooks`` in a protected way at the class level? Unless it's used by other methods why not scope it to ``__init__``?

Reply:
Good catch.